WalletWasabi.Fluent/Controls/AdornerCanvas.cs

```diff
@@ -145,14 +143,17 @@ private void UpdateClip(IControl control, TransformedBounds bounds, bool isEnabl
 		clip.Rect = clipBounds;
 	}
 
-	private void ChildrenCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+	private void ChildrenCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
 	{
 		switch (e.Action)
 		{
 			case NotifyCollectionChangedAction.Add:
-				foreach (Visual i in e.NewItems)
+				if (e.NewItems is { })
```

Comment:
This can also be:

```cs
				if (e.NewItems is { } items)
				{
					foreach (Visual i in items)
					{
						UpdateAdornedElement(i, i.GetValue(AdornedElementProperty));
					}
				}
```